recipe/meta.yaml

```diff
@@ -42,19 +33,42 @@ outputs:
       run:
         - {{ pin_subpackage('beautifulsoup4', max_pin="x.x.x") }}
     test:
+      source_files:
+        - bs4/tests
```

Comment:
We probably don't want to use `source_files` in our tests; conda build's default behavior is to copy these files into the output packages, which among other things, can:
1. dramatically increase the size of the output package; and
2. occasionally trigger security warnings on user systems.

A better approach may be to create a `-tests` package as another output package, then use it as a test requires.

Reply:
I've removed `source_files`